app/src/main/res/layout/activity_main.xml

```diff
@@ -5,44 +5,45 @@
     android:id="@+id/drawer_layout"
     android:layout_width="match_parent"
     android:layout_height="match_parent"
-    android:fitsSystemWindows="true"
-    tools:openDrawer="start">
+    android:fitsSystemWindows="false">
 
-    <android.support.design.widget.CoordinatorLayout
+    <LinearLayout
```

Comment:
I dont understand the need of changes in this file. 

You are just updating the gradle versions, right?

Reply:
I forgot to mention that also updated UI. Please have a look at updated summary and description.